schema.sql

```diff
@@ -1,12 +1,11 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
     id INTEGER GENERATED ALWAYS AS IDENTITY,
     name VARCHAR(100),
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
     weight_kg REAL,
-
+    species VARCHAR,
```

Comment:
- [ ] [Optional] The best way to add a new column is to use `ALTER TABLE`, for this example will be like:
```sql
ALTER TABLE animals ADD species varchar(255);
```